test-packages/e2e-tests/test/mail/mail.spec.ts

```diff
@@ -47,37 +50,48 @@ describe('Mail', () => {
 
 async function sendTestMail(
   connection?: net.Socket,
-  ...mailOptions: Mail.Options[]
-): Promise<void> {
-  // create reusable transporter object using the default SMTP transport
-  const transporter = nodemailer.createTransport({
-    connection,
-    host: 'localhost',
-    port: 5566,
-    // true for 465, false for other ports
-    secure: false,
-    auth: {
-      user: 'user',
-      pass: 'pd'
-    },
-    tls: {
-      // disable tls config to fix the self signed certificate error
-      rejectUnauthorized: false
-    }
-  });
-
-  for (const mailOptionIndex in mailOptions) {
-    // eslint-disable-next-line no-console
-    console.log(`Sending email ${mailOptionIndex}/${mailOptions.length}...`);
-    const response = await transporter.sendMail(mailOptions[mailOptionIndex]);
-    // eslint-disable-next-line no-console
-    console.log(
-      `...email ${mailOptionIndex}/${mailOptions.length} for subject "${mailOptions[mailOptionIndex].subject}" was sent successfully.`
-    );
-  }
-  transporter.close();
-  // eslint-disable-next-line no-console
-  console.log('SMTP transport connection closed.');
+  ...mailOptions: MailOptions[]
+): Promise<MailResponse[]> {
+  const originalProperties = {
+    'mail.smtp.host': 'localhost',
+    'mail.smtp.port': '5566',
+    'mail.user': 'user',
+    'mail.password': 'pd'
+  };
+  const destination: any = {
+    type: 'MAIL',
+    originalProperties
+  };
+  return sendMail(destination, ...mailOptions);
+  // // create reusable transporter object using the default SMTP transport
+  // const transporter = nodemailer.createTransport({
+  //   connection,
+  //   host: 'localhost',
+  //   port: 5566,
+  //   // true for 465, false for other ports
+  //   secure: false,
+  //   auth: {
+  //     user: 'user',
+  //     pass: 'pd'
+  //   },
+  //   tls: {
+  //     // disable tls config to fix the self signed certificate error
+  //     rejectUnauthorized: false
+  //   }
+  // });
```

Comment:
[q] remove comments?

Reply:
I added a todo for cleaning up before merging.